### ISSUE TITLE
Why use  (-1 *F.mse_loss(predict_variable,other_variable)) instead of torch.log(predict_variable)-torch.log(other_variable)

### ISSUE THREAD

**Issue by Marioooooooooooooo** [lich14/CDS#10]
Dear, author: 
CDS is good paper and has a good performance on several super hard maps of SMAC. I am confused about the intrinsic reward is implemented in the code. 
In predict_net.py line 154, log_p_o (represent log p(o_{t+1}|\tau_{t},a_{t})) is calculated by  -1 * F.mse_loss(predict_variable, other_variable, reduction='none'). I want to know why use  -1 * F.mse_loss(predict_variable, other_variable, reduction='none') instead torch.log(predict_variable) for calculating log_p_o?

**Comment by lich14** [comment]
Hi,
That's a good question. In a normal distribution, there are two parameters: the mean and the standard deviation. If you want to estimate both of these parameters to form a normal distribution, you would typically use torch.log() or some other method to calculate log-likelihood. However, in CDS, we assume that the standard deviation is constant. In this case, the log-likelihood becomes -mse_loss(mean, .), where 'mean' is the estimated mean of the normal distribution and '.' represents the observed data."
